packages/reactivity/src/ref.ts

```diff
@@ -101,7 +101,7 @@ function toProxyRef<T extends object, K extends keyof T>(
 // corner case when use narrows type
 // Ex. type RelativePath = string & { __brand: unknown }
 // RelativePath extends object -> true
-type BaseTypes = string | number | boolean
+type BaseTypes = string | number | boolean | Element | Window | Document
```

Comment:
```suggestion
type BaseTypes = string | number | boolean | Node | Window
```

`Node` is a base type that covers `Element`, `Document` and also text/comment nodes.